### PR TITLE
luci: add support for shadowsocks-rust AEAD-2022 ciphers

### PR DESCRIPTION
**Pull request by Gzxhwq**
Wait for shadowsocks-rust 1.15.0 release to support these new ciphers.

### CODE REVIEW

File: luci-app-passwall2/luasrc/model/cbi/passwall2/client/node_config.lua [hunk 1]
```diff
@@ -15,7 +15,8 @@ local ss_encrypt_method_list = {
 
 local ss_rust_encrypt_method_list = {
     "plain", "none",
-    "aes-128-gcm", "aes-256-gcm", "chacha20-ietf-poly1305"
+    "aes-128-gcm", "aes-256-gcm", "chacha20-ietf-poly1305"，
```
Comment (xiaorouji) [inline luci-app-passwall2/luasrc/model/cbi/passwall2/client/node_config.lua:18]: Note symbol.

Comment (Gzxhwq) [reply luci-app-passwall2/luasrc/model/cbi/passwall2/client/node_config.lua:18]: Thank you.

**Comment by Gzxhwq** [comment]
@smallprogram Due to https://github.com/xiaorouji/openwrt-passwall/pull/1990 had been merged. This should be merged too.

**Comment by smallprogram** [comment]
@Gzxhwq I'm not a collaborator of this repository, I don't have permission

**Comment by Gzxhwq** [comment]
> @Gzxhwq I'm not a collaborator of this repository, I don't have permission

OK

@xiaorouji

**Pull request merged**